### ISSUE TITLE
WDIO Appium Cucumber project not running on MacOS M1

### ISSUE THREAD

**Issue by ken4ward** [webdriverio/webdriverio#12203]
There's an issue being experienced on M1 for a project that runs fine on Intel MacOs machines. If a user tries to run it, it indefinitely gets stuck. 

Log
 ```
project % npm run en:android

> project@1.0.0 en:android
> wdio run config/env/dev/android/en.conf.android.js

This is test definitions = /Users/.../WebstormProjects/jsmobsframework/cucumber/step_definitions/*.js

Execution of 2 workers started at 2024-02-09T09:35:30.626Z

2024-02-09T09:35:30.632Z DEBUG @wdio/utils:initializeServices: initialize service "appium" as NPM package
2024-02-09T09:35:30.635Z INFO @wdio/cli:launcher: Run onPrepare hook
2024-02-09T09:35:30.636Z INFO @wdio/appium-service: Will spawn Appium process: appium --base-path / --relaxed-security --port 4724 --log ./appium.log --debug-log-spacing
```

It does not go pass this. The same project runs fine on Intel machines.

**Comment by erwinheitzman** [comment]
Hi @ken4ward, you seem to create issues before asking questions in discord. Please do it the other way around next time as again we have talked on Discord already.

**Comment by erwinheitzman** [comment]
@ken4ward is this still an issue?

**Comment by christian-bromann** [comment]
Closing. Happy to re-open if there are remaining issues.

**Comment by ken4ward** [comment]
This is still an issue. I am using WDIO version 8.29.0. I started the project on MacOS Intel, and exported it to M1. It runs fine on Intel but didn't work on M1. Please help to resolve.

**Comment by christian-bromann** [comment]
> It runs fine on Intel but didn't work on M1. Please help to resolve.

Can you please provide a reproducible example? Thanks!

**Comment by erwinheitzman** [comment]
Can you please create a minimal reproducible example @ken4ward ? We need it to debug this

**Comment by ken4ward** [comment]
I have added @christian-bromann and @erwinheitzman to the repository. https://github.com/ken4ward/jsmobsframework.git

**Comment by erwinheitzman** [comment]
The node_modules are in the repo, have you deleted the node_modules directory and ran npm install?

**Comment by ken4ward** [comment]
I'm doing that again. I should give feedback in the net 10 minutes.

**Comment by christian-bromann** [comment]
I was able to reproduce the issue. It seems like we are only getting the first bytes of the stderr with the content `dbug` which maps to the Appium output when running it directly, and nothing more. Very strange. @jlipps any ideas?

**Comment by ken4ward** [comment]
Is it right if I add you to the repo, @jlipps ?

**Comment by ken4ward** [comment]
Yea, I should strip it. I would have made it open, but it's a project created for the company i work with, so it seems I have less control on its visibility. Though I provided this project, it seems the issue has less or nothing to do with it. As @christian-bromann stated, if you go by the version I used in the project to create a new project, it does not run on M1 systems. The earlier versions of wdio do not have this issue.
